### PR TITLE
Clean up treemap in web UI

### PR DESCRIPTION
**Pull request by mofojed**
- Automatically fill in labels with IDs if labels are blank
- Fix a possible null axis check

Tested with https://github.com/deephaven/deephaven-core/pull/2678
Was able to open and interact with Treemaps correctly

### CODE REVIEW

File: packages/chart/src/FigureChartModel.js [hunk 1]
```diff
@@ -566,6 +566,13 @@ class FigureChartModel extends ChartModel {
       if (yLow && yHigh && yLow !== y) {
         seriesData.error_y = ChartUtils.getPlotlyErrorBars(y, yLow, yHigh);
       }
+    } else if (plotStyle === dh.plot.SeriesPlotStyle.TREEMAP) {
+      const { ids, labels } = seriesData;
+      if (ids && !labels) {
```
Comment (vbabich) [inline packages/chart/src/FigureChartModel.js:571]: What's the type of `labels`? If it's an array, is it enough to only check for truthy/falsy value here?

Comment (mofojed) [reply packages/chart/src/FigureChartModel.js:571]: Yes, it's an array. You're right I should be checking for undefined instead.

Comment (vbabich) [reply packages/chart/src/FigureChartModel.js:571]: Actually, I was thinking about the situation when the `labels` array is truthy but empty. Is it possible we get an empty array from the API?

Comment (mofojed) [reply packages/chart/src/FigureChartModel.js]: Right - if we get an empty array, we should be keeping that. It's _possible_, but would also mean there's nothing to plot. Basically means you created a plot with an empty data set.

**Review by vbabich — APPROVED** [review]

**Pull request merged**